src/map/utils/synthutils.cpp

```diff
@@ -345,8 +345,8 @@ namespace synthutils
                 case 3: // 1 in 4
                     chanceHQ = 25.0;
                     break;
-                case 2: // 1 in 20
-                    chanceHQ = 5.0;
+                case 2: // 1 in 16
+                    chanceHQ = 6.25;
```

Comment:
More than happy to split this into a different PR or keep it the same as it has been.

Reply:
I would personally change it because the data shows it closer to the rates SE use for other things but if xaver wants to keep it 5 we can open a discussion post and gather all the data in one place before making a change like this one. (if at all)

Reply:
Lets not mix stuff. I would focus this PR on the single multiplier setting.

As for the rates, feel free to open a discussion, but as I said up there, without really really solid evidence, they should stay against player's odds, for server OPs to judge.

Reply:
Removed.

Reply:
Added a discussion: https://github.com/LandSandBoat/server/discussions/5720